x/tss/abci.go

```diff
@@ -117,23 +117,17 @@ func emitSignStartEvent(ctx sdk.Context, k types.TSSKeeper, voter types.InitPoll
 			nonParticipantShareCounts = append(nonParticipantShareCounts, validator.ShareCount)
 			continue
 		}
-
-		// metrics for sign participation
-		metrics.SetGaugeWithLabels([]string{types.ModuleName, "sign", "participation"}, 0,
-			[]metrics.Label{
-				telemetry.NewLabel("sigID", info.SigID),
-				telemetry.NewLabel("address", validator.GetSDKValidator().GetOperator().String()),
-				telemetry.NewLabel("share_count", strconv.FormatInt(validator.ShareCount, 10)),
-				telemetry.NewLabel("timestamp", strconv.FormatInt(ctx.BlockTime().Unix(), 10)),
-				telemetry.NewLabel("block", strconv.FormatInt(ctx.BlockHeight(), 10)),
-			})
 	}
 
 	// no need to check if these exists again, sanity checks for that passed at this point
 	key, _ := k.GetKey(ctx, info.KeyID)
 	keyType := k.GetKeyType(ctx, info.KeyID)
 	keyRequirement, _ := k.GetKeyRequirement(ctx, key.Role, keyType)
 
+	// sign participation
+	telemetry.SetGaugeWithLabels([]string{types.ModuleName, "sign", "participation"},
+		float32(len(k.GetSignParticipants(ctx, info.SigID))), []metrics.Label{telemetry.NewLabel("keyID", string(info.KeyID))})
+
```

Comment:
is this still going to bloat based on the key ID?

Reply:
Possibly, but keygen participation was also using keyID as a label, yet apparently wasn't causing any issue. So I decided to preserve keyID for sign participation, and remove keyID from keygen participation. Nonetheless, if we find that sign participation metrics still result in bloat, we can completely remove them in a patch without having a consensus-breaking change.

Reply:
why not remove it now? 😅

Reply:
Since it seems unanimous that these should be removed, I did just that